### ISSUE TITLE
Pytorch Transformers: should I re-initialize my optimizer and my scheduler before I try to fine tune my neural network on the different dataset?

### ISSUE THREAD

**Issue by h56cho** [pytorch/text#659]
Hello,

I am doing NLP, and I have this block of Transformer body that was already trained on dataset A.
Now I am interested in fine tuning this same Transformer on a new dataset B.

In my Python code, should I re-initialize my optimizer and my scheduler before I try to fine tune my neural network on the different dataset? or is this not needed?

Thank you,
